docs/version0.46.md

```diff
@@ -40,6 +41,12 @@ To learn more about support for OpenJ9 releases, including OpenJDK levels and pl
 
 OpenSSL native cryptographic support is added for the MD5 message digest algorithm, providing improved cryptographic performance. OpenSSL support is enabled by default. If you want to turn off support for the MD5 message digest algorithm, set the [`-Djdk.nativedigest`](djdknativedigest.md) system property to `false`.
 
+### Support added for the `com.sun.management.ThreadMXBean.getTotalThreadAllocatedBytes()` API
+
+With this release, the OpenJ9 VM implementation supports measurement of the total memory allocation for all threads (`com.sun.management.ThreadMXBean.getTotalThreadAllocatedBytes()` API).
+
+The `getTotalThreadAllocatedBytes()` method now returns true instead of false and does not throw the "UnsupportedOperationException" error.
```

Comment:
It doesn't return true, it returns a `long` value which represents the total thread allocated bytes, rather than returning -1.

Reply:
Prior to 0.41 it would have thrown UnsupportedOperationException, but from 0.41 which added support for thread allocated bytes, it returned -1.